### ISSUE TITLE
Incorrect font size for custom font

### ISSUE THREAD

**Issue by Zylatis** [vega/vega-lite#5606]
I am using the following code (link to Vega Lite editor, code generated with the help of Altair)

https://tinyurl.com/w4ob663

which uses the non-standard font Museo Sans 300. The font appears to display correctly but the font size doesn't seem to work - rather than scaling the font it seems to just adjust the surrounding padding.

I have successfully used this font in D3.js code, importing the font info from an external css sheet. However even importing this css sheet in the html used to render this chart doesn't help.

It seems potentially related to the issue here
https://github.com/vega/vega-lite/issues/4657

but i'm not sure how to leverage that (react-focused) solution here.

Any/all help appreciated

**Comment by domoritz** [comment]
I don't have your font installed but I get this.

<img width="837" alt="Screen Shot 2019-12-01 at 17 03 47" src="https://user-images.githubusercontent.com/589034/69923856-93f3da00-145c-11ea-9704-a415720ff60d.png">

What output are you getting? What are you expecting? Can you reproduce the problem with a standard font?

**Comment by Zylatis** [comment]
Thanks @domoritz for the quick reply!

If i run locally or use that Vega-renderer link I get the following changing the font sizes from 30 to 60

![img1](https://user-images.githubusercontent.com/1456631/69923919-4464eb00-14fc-11ea-9e4f-dc83f0a710c3.png)
![img2](https://user-images.githubusercontent.com/1456631/69923922-45961800-14fc-11ea-882e-eb55c91730b3.png)
All that happens is the padding seems to increase (these are screenshots taken from that Vega renderer website but it matches what i observe locally)

**Comment by domoritz** [comment]
I can't reproduce this issue with a few of the fonts I tried on my machine (including non-standard fonts). 

Can you try the cross product of different browsers, renderers (svg and canvas), and a few different fonts and report what you find?

**Comment by Zylatis** [comment]
Thanks @domoritz but on closer inspection I think it is still not loading the correct font on yours, e.g. compare the 't' and '1' in this to your screenshot
![font](https://user-images.githubusercontent.com/1456631/69926628-46817680-1509-11ea-9297-23b4ece877a8.png)

I think probably it is just defaulting to a different font in the background?

**Comment by domoritz** [comment]
Yes. I don't have `Museo Sans` installed so the browser defaults to `sans`. To me, it seems as though Vega does not use the font correctly to determine the space needed. I'm moving the issue.
